aea/configurations/schemas/aea-config_schema.json

```diff
@@ -50,6 +50,15 @@
         }
       }
     },
+    "connection_private_key_paths": {
```

Comment:
copy-pasted from `private_key_path`. That is:
```
connection_private_key_path:
  fetchai: path/to/fetch_private_key.txt
  ethereum: path/to/ethereum_private_key.txt
```

It is optional, so we are backward-compatible.